### DISCUSSION TITLE
Request for Refund

### DISCUSSION THREAD

**Discussion by RUO08310130** [aclap-dev/video-downloadhelper#340] [Help and Q&A]
### Your question

I recently purchased a license for DownloadHelper, expecting it to facilitate downloading videos from Udemy. Unfortunately, it doesn't support this function.

As Udemy courses are my primary learning resource, I need a solution that can download these videos. Therefore, I kindly request a refund for the DownloadHelper license.

Thank you for your understanding. Please advise on the refund process.

**email** - ruo@stolt.com

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: accepted
key (only 16 first characters): e626a454119c483f
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\ruben",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
hi,
Sorry I did not find a purchase with the email address. Can you please resend the details of your purchase at licenses@downloadhelper.net?

**Reply by RUO08310130** [reply]
apologies, the email address is rubenncls@gmail.com, i am done sending the purchase details to licenses@downloadhelper.net. Thank you

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.
Regards,
jerome

**Reply by bookaya619hbk** [reply]
Hi, I have purchased the downloadhelper license for Google Chrome with this email address: tushar.version1@gmail.com to download YouTube videos.  Hence, DownloadHelper doesn't support this anymore. I would like to request a refund on my purchase.

**Reply by jcfrog** [reply]
Hi,
Sorry, refund is only possible the 1st month

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I am just another user.  Also, please read #2214.

I would like to point out that VDH for Chrome has NEVER worked on YouTube.  This is a restriction imposed by Google, the company that owns both the YouTube web site & the Chrome browser.  This is well known & last time I looked, it is clearly marked in the VDH entry on the Google web store.

**Reply by GrampaWildWilly** [reply]
When you go into our Wiki & look to install VDH for Chrome, it takes you to this URL:

https://chromewebstore.google.com/detail/video-downloadhelper/lmjnegcaeklhafolokijcfjliaokphfk

On that page, you will find this:

![#01](https://github.com/user-attachments/assets/36356763-c1a5-4957-9673-0507cc925a52)

There's no excuse for anybody not knowing that VDH for Chrome does not work on YouTube.

**Comment by bookaya619hbk** [comment]
Can you please transfer my Google Chrome license to Firefox if possible.

**Reply by jcfrog** [reply]
Sorry, it's not possible. Chrome and Firefox premium are separate products with different prices and features.

**Comment by bookaya619hbk** [comment]
Hi
I didn`t read the documentation. I think before Manifest 3 version I was
able to download youtube videos from download helper but after the manifest
3 update. I am not able to download any videos either from IDM or
downloadhelper.

**Comment by wedmasterblog-crypto** [comment]
Hello, I purchased the DownloadHelper license for Google Chrome using this email address: johanlinares380@gmail.com. to download videos from Platzi. but, DownloadHelper isn't compatible. I'd like to request a refund for my purchase.

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by kariot731** [comment]
Hey, i purchased the license for VDH in mozilla and it worked  fine at first, however now whenever i download audio files either the audio file length doesnt match the file i downloaded, or the video doesnt download at all which happens most of the time. I get this message :"Sadly the addon encountered an unexpected error. It would help a lot if you could report that error (it's anonymous!).  "

With this all in mind i would like to request a refund as the addon it is not working as intended and I cant get any use from it.

my email used to purchase was: kardojobb@gmail.com

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by rodk92** [comment]
Hello, I would like a refund because I ended up purchasing the extension for Chrome instead of Firefox, believing that both would be able to download YouTube videos. 
The email address used is rodkb92@gmail.com. 
Thank you.

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by adfrex1** [comment]
Hey there I would like a refund I emailed you

**Comment by zaynmalyk66-hub** [comment]
Hi
I had purchased your premium package last year for life time. Now you have again deducted the amount. Kindly fix this problem and refund ihe amount as soon as possible.
Thanks

**Reply by jcfrog** [reply]
We never do that.
I sent you an email.

**Comment by dcmarvelfanboy1999-code** [comment]
Hello, I purchased the Video DownloadHelper license for Google Chrome yesterday using this email address: alonckilci@gmail.com

I would like to request a refund.

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.
